crates/node/builder/src/launch/common.rs

```diff
@@ -504,6 +505,7 @@ where
                     target_triple: VERGEN_CARGO_TARGET_TRIPLE,
                     build_profile: BUILD_PROFILE_NAME,
                 },
+                ChainSpecInfo { name: format!("{}", self.left().config.chain.chain) },
```

Comment:
same as above

Reply:
Fixed in https://github.com/paradigmxyz/reth/pull/10737/commits/e8a877133a58ff5730ebbcc61cba5dd784eb118b